network-api/networkapi/templates/pages/buyersguide/catalog.html

```diff
@@ -2,7 +2,7 @@
 
 {% load static env i18n static wagtailimages_tags cache bg_nav_tags localization %}
 
-{% block bodyclass %}pni catalog{% endblock %}
+{% block bodyclass %}pni catalog tw-pt-[150px]{% endblock %}
```

Comment:
Same. Could you add a comment explaining why we need a 150px padding here?

Reply:
Answered in comment above